### PR TITLE
remove TestCrateSummary::Interrupted and don't print Skipped

### PR DESCRIPTION
**Pull request by tkumor3**
Closes #

## Introduced changes

-
-

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code
- [x] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: crates/forge/src/test_case_summary.rs [hunk 1]
```diff
@@ -41,17 +41,7 @@ pub enum TestCaseSummary {
         name: String,
     },
     /// Test case skipped due to exit first
```
Comment (cptartur) [inline crates/forge/src/test_case_summary.rs:43]: Is this only possible reason for `Skipped` status now?

Comment (tkumor3) [reply crates/forge/src/test_case_summary.rs:43]: In general, it's returned when we do not care about the test result, in the case when there is no test that returns Err we use it to know how many tests was skipped

Comment (cptartur) [reply crates/forge/src/test_case_summary.rs:43]: I'd update this comment then

**Review by MaksymilianDemitraszek — APPROVED** [review]
Please update the comment about skipped before merging.

**Pull request merged**